.storybook/preview.js

```diff
@@ -26,10 +26,11 @@ export const decorators = [
 
 export const globalTypes = {
   bedrock: {
-    name: 'bedrock',
     description: 'CSS Bedrock',
     defaultValue: 'enabled',
     toolbar: {
+      // empty string to omit name from toolbar
+      title: '',
       icon: 'link',
       items: [
         { value: 'enabled', title: 'Bedrock enabled' },
```

Comment:
Please compare with https://github.com/zendeskgarden/react-containers/blob/3676836c2463bee4163139299f156bffa7d26a39/.storybook/preview.js#L59-L69 which seems to be working as intended in v7.